### ISSUE TITLE
Add guidance for contributors who want to help with the PR backlog

### ISSUE THREAD

**Issue by shannonxtreme** [kubernetes/website#46141]
We should have guidance for new contributors and org members for how to filter through the project's PR backlog and contribute towards reducing it. This guidance should cover the following points (thanks @sftim):

1. Our goal to have &le; 100 open English PRs in the project
1. New contributors and org members:
    1. Filter for PRs that:
        - Don't have an LGTM label
        - Are in the contributor's preferred language
        - Don't already have a _useful review_ left by someone else. Useful meaning substantial - if the previous review is typos or nits, worth looking into.
    1. Sort by oldest first and work forwards in time
1. Reviewer role:
   1. Link off to role responsibilities

This content should probably go into  https://kubernetes.io/docs/contribute/review/reviewing-prs/ and potentially be linked to from https://github.com/kubernetes/community/blob/master/contributors/guide/review-guidelines.md (although that's for all contributors and not just docs)

/sig contributor-experience
/sig docs

**Comment by sftim** [comment]
> This content should probably go into https://kubernetes.io/docs/contribute/review/reviewing-prs/ and potentially be linked to from https://github.com/kubernetes/community/blob/master/contributors/guide/review-guidelines.md (although that's for all contributors and not just docs)

Can we put it into https://k8s.dev/docs/ and link there? It'll be English-only, but that's OK: everyone who works on PRs should have at least some familiarity with English.

**Comment by divya-mohan0209** [comment]
I agree with @sftim's point since filtering and contributing also fit into the scope of the wider project. Amending k8s.dev & linking it somewhere in our contributing docs should help.

**Comment by natalisucks** [comment]
@shannonxtreme Can you please revise the following guidance:

> Our goal to have ≤ 100 open PRs in the project

Our goal is to have approximately 100 PRs open for _English language and site-wide changes_. This is an important distinction because we have a lot of localization PRs that aren't under the purveiw of PR wrangling, but are rather the domain of localization teams, and their resources are decidely smaller the than SIG-wide reveiwers pool.

**Comment by shannonxtreme** [comment]
Thanks Natali, added. @divya-mohan0209 @sftim for the k8s.dev one, I was looking at that and it didn't seem like we had a "guidelines for PR reviews" type document there. That's why I ended up in the k/community repo, which does have that guidance. Do you think that it's worth creating a new page in k8s.dev/docs instead?

**Comment by priyanshsao** [comment]
can i work on this issue?

**Comment by shannonxtreme** [comment]
Yes @priyanshsao but maybe let's get confirmation in these comments about where exactly to put this information before you do a PR!

**Comment by divya-mohan0209** [comment]
@shannonxtreme : Good point! I would suggest that it be added as a section into [the pull requests guide](https://www.kubernetes.dev/docs/guide/pull-requests/) on the k8s.dev site since there's already information that is very similar to the information that we're trying to put forth. Thoughts?

**Comment by shannonxtreme** [comment]
@divya-mohan0209 That page seems to be tightly focused on submitting a PR, not reviewing one. What about https://www.kubernetes.dev/docs/guide/contributing/#code-review ? There's a section there for reviewers about the expectations for them. Maybe a subsection under Code review for PR reviewer guidelines?
